### PR TITLE
EVG-13928 Display tickets that were filed using a web-hook

### PR DESCRIPTION
**Pull request by malikchaya2**
image

### CODE REVIEW

**Comment by SupaJoon** [comment]
@malikchaya2 do you know a url in one of the spruce environments where these code changes are executed so I can run them locally?

**Comment by SupaJoon** [comment]
evergreen-retry

**Comment by SupaJoon** [comment]
evergreen retry

**Comment by malikchaya2** [comment]
> @malikchaya2 do you know a url in one of the spruce environments where these code changes are executed so I can run them locally?

This taskId in staging: 
5ffe393097b1d3759dd3c1a3_release_test_release_aee6aa9bef8c5d454835584e3ff34573c9feb660_21_01_11_21_51_47

**Review by SupaJoon — APPROVED** [review]

**Comment by khelif96** [comment]
can you merge main. we replaced our queries with `*.graphql` files

File: src/pages/task/taskTabs/buildBaronAndAnnotations/BBFIleTicket.tsx [hunk 1]
```diff
@@ -14,8 +14,8 @@ import { ButtonWrapper } from "./BBComponents";
 interface BBFileTicketProps {
```
[the remark this answers is not in the document]
Comment (malikchaya2) [reply src/pages/task/taskTabs/buildBaronAndAnnotations/BBFIleTicket.tsx:14]: good catch! thanks!

File: src/pages/task/taskTabs/buildBaronAndAnnotations/AnnotationTicketsTable.tsx [hunk 2]
```diff
@@ -224,6 +224,48 @@ export const AnnotationTicketsTable: React.FC<Props> = ({
...
+        <Table
+          tableLayout="fixed"
+          data-test-id="created-issues-table"
+          dataSource={createdIssues}
+          rowKey={({ issueKey }) => issueKey}
+          columns={columns}
+          pagination={false}
+          showHeader={false}
+        />
+      </ErrorBoundary>
```
[the remark this answers is not in the document]
Comment (malikchaya2) [reply src/pages/task/taskTabs/buildBaronAndAnnotations/AnnotationTicketsTable.tsx:263]: copy paste. I can remove it.

File: src/pages/task/taskTabs/buildBaronAndAnnotations/BBFileTicket.tsx [hunk 3]
```diff
@@ -14,8 +14,8 @@ import { ButtonWrapper } from "./BBComponents";
 interface BBFileTicketProps {
   taskId: string;
   execution: number;
-  setCreatedTicketsCount: React.Dispatch<React.SetStateAction<number>>;
-  createdTicketsCount: number;
+  setCreatedTicketsCount?: React.Dispatch<React.SetStateAction<number>>;
+  createdTicketsCount?: number;
 }
 
 export const BBFileTicket: React.FC<BBFileTicketProps> = ({
```
Comment (khelif96) [inline src/pages/task/taskTabs/buildBaronAndAnnotations/BBFileTicket.tsx:21]: Is this component necessary it doesn't do anything but wrap  `<FileTicket />` ? We should either remove it or remove `<FileTicket />` and move the logic over.

File: src/pages/task/taskTabs/buildBaronAndAnnotations/BuildBaronContent.tsx [hunk 4]
```diff
@@ -49,63 +48,66 @@ export const BuildBaronContent: React.FC<BuildBaronCoreProps> = ({
   return (
     <span data-cy="bb-content">
       {loading && <Skeleton active title={false} paragraph={{ rows: 4 }} />}
-      {bbData && (
-        <>
+      <>
```
Comment (khelif96) [inline src/pages/task/taskTabs/buildBaronAndAnnotations/BuildBaronContent.tsx:51]: Is this fragment necessary?

File: src/pages/task/taskTabs/buildBaronAndAnnotations/BuildBaronContent.tsx [hunk 5]
```diff
@@ -49,63 +48,66 @@ export const BuildBaronContent: React.FC<BuildBaronCoreProps> = ({
   return (
     <span data-cy="bb-content">
```
Comment (khelif96) [inline src/pages/task/taskTabs/buildBaronAndAnnotations/BuildBaronContent.tsx:49]: This should be a `<div/>` https://www.tutorialspoint.com/What-is-the-difference-between-HTML-tags-div-and-span#:~:text=The%20element%20shows%20the,of%20text%2C%20images%2C%20etc.

**Review by khelif96 — CHANGES_REQUESTED** [review]

File: src/pages/task/taskTabs/buildBaronAndAnnotations/BBFileTicket.tsx (hunk 3 above)
Comment (malikchaya2) [reply src/pages/task/taskTabs/buildBaronAndAnnotations/BBFileTicket.tsx:21]: Ah yes, it probably evolved to a point where it is no longer necessary. Removing it.

File: src/pages/task/taskTabs/buildBaronAndAnnotations/BuildBaronContent.tsx (hunk 4 above)
Comment (malikchaya2) [reply src/pages/task/taskTabs/buildBaronAndAnnotations/BuildBaronContent.tsx:51]: removing it.

**Review by khelif96 — APPROVED** [review]

**Pull request merged**